db/schema.sql

```diff
@@ -19,6 +19,10 @@ CREATE TABLE adoptable_creatures (
     type_id INTEGER REFERENCES creature_types(id) NOT NULL
 );
 
+<<<<<<< HEAD
```

Comment:
please resolve merge conflicts and remove merge markers prior to submitting PRs.